src/snapshots/fetch.h

```diff
@@ -42,23 +42,31 @@ namespace snapshots
 
   static std::optional<SnapshotResponse> try_fetch_from_peer(
     const std::string& peer_address,
-    const std::string& path_to_peer_cert,
+    const std::string& path_to_peer_ca,
```

Comment:
Minor rename, because I had a panic that this was a node cert and would fail if we redirected to a different node. It's not - its a service cert, so is actually the CA for any peers we might speak to.